src/NuGet.Core/NuGet.Packaging/Core/NuspecUtility.cs

```diff
@@ -114,8 +119,62 @@ public static bool IsServiceable(XElement metadataNode)
                 return false;
             }
 
-            string value = element.Value ?? element.Value.Trim();
+            var value = element.Value ?? element.Value.Trim();
             return System.Xml.XmlConvert.ToBoolean(value);
         }
+
+        /// <summary>
+        /// Gets the FrameworkReference groups. This refers to the FrameworkReference concept added in .NET Core 3.0
+        /// </summary>
+        /// <param name="metadataNode">The metadata node</param>
+        /// <param name="frameworkProvider">A FrameworkNameProvider</param>
+        /// <param name="useMetadataNamespace">Whether or not to use the metadata element's namespace when getting the framework reference nodes.
+        /// If false is specified, only the local names of the FrameworkReference nodes are used
+        /// for comparison. If true is specified, the fremeworkreference nodes must have the same
+        /// namespace as the metadata node.</param>
+        /// <returns></returns>
+        internal static IEnumerable<FrameworkSpecificGroup> GetFrameworkReferenceGroups(XElement metadataNode, IFrameworkNameProvider frameworkProvider, bool useMetadataNamespace)
+        {
+            var ns = useMetadataNamespace ? metadataNode.GetDefaultNamespace().NamespaceName : null;
```

Comment:
>ns [](start = 16, length = 2)

`ns` -> `namespace`?

Reply:
It'll have to be `@namespace` :D 

As it's only a local var, I'd rather not, as I don't see it adding too much value.